vignettes/base-vs-fs.Rmd

```diff
@@ -0,0 +1,57 @@
+| `dir.create("/new-path"); file.copy("/path", "/new-path", recursive=TRUE)` | `dir_copy("/path", "/new-path")`          | `cp /path /new-path`               |
+| `dir.create("/path")`                                                      | `dir_create("/path")`                     | `mkdir /path`                      |
+| `unlink("/path", recursive = TRUE)`                                        | `dir_delete("/path")`                     | `rm -rf /path`                     |
+| `dir.exists("/path")`                                                      | `dir_exists("/path")`                     | `if [ -d "/path" ]; then ... ; fi` |
+| `file.rename("/path", "/new-path")`                                        | ~~`dir_move()`~~... (see `fs::file_move`) | `mv /path /new-path`               |
+
+
+### File functions
+
+| base                                | fs                                          | shell                              |
+|-------------------------------------+---------------------------------------------+------------------------------------|
+| *No easy way*                       | `file_chmod("/path", "mode")`               | `chmod mode /path`                 |
+| *No easy way*                       | `file_chown("path", "user_id", "group_id")` | `chown options /path `             |
+| `file.copy("/path", "/new-path")`   | `file_copy("/path", "/new-path")`           | `cp /path /new-path`               |
+| `file.create("/new-path")`          | `file_create("/new-path")`                  | `touch /new-path`                  |
+| `unlink("/path")`                   | `file_delete("/path")`                      | `rm /path`                         |
+| `file.exists("/path")`              | `file_exists("/path")`                      | `if [ -f "/path" ]; then ... ; fi` |
+| `file.info("/path")`                | `file_info("/path")`                        | `ls -al /path`                     |
+| `file.rename("/path", "/new-path")` | `file_move("/path", "/new-path")`           | `mv /path /new-path`               |
+| `browseURL("path")`                 | `file_show("/path")`                        | `open /path`                       |
+| *No easy way*                       | `file_touch()` (**doesn't create file**)    | `touch /path`                      |
+
+### Path functions
+
+| base                                             | fs                                                   | shell                         |
+|--------------------------------------------------+------------------------------------------------------+-------------------------------|
+| `file.path("top_dir", "nested_dir", "file.ext")` | `path("top_dir", "nested_dir", "file", ext = "ext")` | `top_dir/nested_dir/file.ext` |
+| `path.expand()`                                  | `path_expand("~/path")`                              | *Need scripting.*             |
```

Comment:
maybe the shell one could be `realpath -m -s ~/path`. realpath is a GNU specific thing, but seems reasonable to use here.

Reply:
It would also be nice to include `path_real()` and `path_norm()` as additional entries here, but I can do that later if needed.

Reply:
Added `realpath -m -s ~/path` (could not try it on this machine).
Maybe it is better that you add `path_real()` and `path_norm()`. I am not sure what would be the **fs** and **shell** short equivalents.